oasis-webapp/src/main/java/oasis/tools/Bootstrap.java

```diff
@@ -362,7 +374,18 @@ private String createDatacore(String oasisOrgId, String adminAccountId) {
     scope.getName().set(ULocale.ROOT, "Datacore");
     scopeRepositoryProvider.get().createOrUpdateScope(scope);
 
-    // XXX: do we need a service?
+    Service service = new Service();
+    service.setLocal_id("playground");
+    service.setInstance_id(instance.getId());
+    service.setProvider_id(instance.getProvider_id());
+    service.setVisibility(Service.Visibility.HIDDEN);
+    service.setAccess_control(Service.AccessControl.ANYONE);
```

Comment:
Currently, the Playground is only accessible to members of the Ozwillo organization.